src/omnisharp/server.ts

```diff
@@ -235,20 +235,17 @@ export class OmniSharpServer {
 
         const solutionPath = launchTarget.target;
         const cwd = path.dirname(solutionPath);
+        this._options = Options.Read();
+        
         let args = [
             '-s', solutionPath,
             '--hostPID', process.pid.toString(),
             '--stdio',
             'DotNet:enablePackageRestore=false',
-            '--encoding', 'utf-8'
+            '--encoding', 'utf-8',
+            '-loglevel', this._options.loggingLevel
         ];
 
-        this._options = Options.Read();
-
-        if (this._options.loggingLevel === 'verbose') {
-            args.push('-v');
-        }
```

Comment:
Seems like an easy thing to do, and it's consistent with other changes to options in the extension. Just implement a mapping from 'verbose' to 'debug' in Options.read().